src/app/modules/main/chat_section/module.nim

```diff
@@ -1,5 +1,6 @@
 import NimQml, Tables, chronicles, json, sequtils
 import io_interface
+import model as chats_model
```

Comment:
@osmaczko this `model` was already imported in the line num 5. Please revert this line.

Reply:
I removed `model` from line num 5, otherwise, there is an ambiguity error with `contacts_model.Model`